Library/Homebrew/extend/ARGV.rb

```diff
@@ -88,8 +88,9 @@ def next
   end
 
   def value(arg)
-    arg = find { |o| o =~ /--#{arg}=(.+)/ }
-    $1 if arg
+    return if arg.empty? || arg.index("-") || arg.index("=")
+    match = find { |opt| opt.start_with?("--#{arg}=") }
```

Comment:
Tiny nit but generally I've seen `match` used for Regex objects so might be worth another name? Feel free to ignore, though.